config/crd/bases/skupper_router_access_crd.yaml

```
@@ -10,38 +10,86 @@ spec:
       storage: true
       schema:
         openAPIV3Schema:
-          description: "Configuration for secure access to the site router"
+          description: |-
+            Configuration for secure access to the site router.  The
+            configuration includes TLS credentials and router ports.  The
+            RouterAccess resource is used to implement link access for sites.
           type: object
           properties:
             spec:
               type: object
               properties:
                 roles:
+                  description: |-
+                    The named interfaces by which a router can be accessed.  These
+                    include "inter-router" for links between interior routers and
+                    "edge" for links from edge routers to interior routers.
                   type: array
                   items:
                     type: object
                     properties:
                       name:
+                        description: The role name. Either "inter-router" or "edge".
                         type: string
                       port:
+                        description: The port for the router to bind. Must not conflict with another role.
                         type: integer
                     required:
                       - name
                 generateTlsCredentials:
+                  description: |-
+                    When set, Skupper generates the TLS credentials to be
+                    stored in the Secret specified by `tlsCredentials`. See
+                    also `issuer`.
                   type: boolean
                 issuer:
                   type: string
+                  description: |-
+                    The name of the Kubernetes Secret containing the signing CA
+                    used to generate TLS certificates for the RouterAccess when
+                    `generateTlsCredentials` is set.
                 accessType:
+                  description: |-
+                    Configures the access type for the router endpoints.
+                    Available access types and the default selection is
+                    configured on the Skupper controller for Kubernetes.
+
+                    The options available by default are:
+                      `local`: No external ingress. Implies a Kubernetes Service with type CluterIP.
```

```suggestion
                    - `local`: No external ingress. Implies a Kubernetes Service with type CluterIP.
```
bullets for all items in list